.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
-svg/output/
\ No newline at end of file
+svg/output/
+.idea/
+npm-debug.log
```

Comment:
I wouldn't be opposed, but thought it would be handy for testing output from previous builds. The project is also a little strange at the movement since we currently check in any changes after running `grunt`